dawn/test/unit-test/dawn4py/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
We created dawn4py-tests to avoid a problem with dawn4py. Should these go in there?